python/hidet/graph/tensor.py

```diff
@@ -1468,3 +1510,26 @@ def asarray(obj, /, *, dtype=None, device=None) -> Tensor:
             array = array.astype(np.float32)
         ret = from_numpy(array)
     return ret.to(dtype=dtype, device=device)
+
+
+def cat(tensors: List[Tensor], dim: int):
+    """
+    Concatenates the list of Tensors along dim.
+
+    Requires
+    --------
+    All shapes for the list of tensors are equal except for dim.
+    """
+    from .ops import concat
+
+    return concat(tensors, dim)
+
+
+def matmul(a: Tensor, b: Tensor) -> Tensor:
+    """
+    Performs matrix multiplication between a and b
+
+    """
+    from .ops import matmul as _matmul
+
+    return _matmul(a, b)
```

Comment:
Hi @Aalanli , is it necessary to place them here? I prefer we leave the definitions of these two functions to `hidet.ops.*`, and directly use `hidet.ops.concat` and hidet.ops.matmul`.

Reply:
Ok, will change.

Reply:
Thanks, just want to keep the `hidet.graph.tensor` namespace clean.